cli/cmd/cluster.go

```diff
@@ -133,14 +137,34 @@ func GenerateOutput(cmd *command, clusterData *ClusterData) error {
 	return errors.New(response)
 }
 
-func doSetup() (*ClusterData, error) {
+func GetClusterDataAssertOnCluster(connection *dbconn.DBConn) (*ClusterData, error) {
```

Comment:
```suggestion
func getClusterDataAssertOnCluster(connection *dbconn.DBConn) (*ClusterData, error) {
```
seems the existing convention is for lower-cased first letter ?

Reply:
Is this function going to public or private? If it will be private, I agree with this change.

Reply:
Is uppercase the expectation for public functions?

Reply:
In Go, uppercase functions/variables will be public and lowercase will be private. It's their method to get around using `public` and `private` keywords.